#goal definition
string direction
float32 duration
float32 speed
---
#result definition
string message
---
#feedback definition
float32 time_left
